### ISSUE TITLE
Installation doesn't work

### ISSUE THREAD

**Issue by MomoInSpace** [jczhang02/luasnips-mathtex-snippets#1]
I tried to install it as described with packer. It installs, but I can't use the snippets. The problem is that the setup function in the init.lua file isn't called. It is not clear to me how to call it after installing it with packer.
My setup can be found in the 'test-mathtex' branch in:
https://github.com/MomoInSpace/nvim-tex

**Comment by jczhang02** [comment]
Sorry for delay reply. Can this issue be closed?

**Comment by MomoInSpace** [comment]
No, sorry. You can check out the branch ^^, it sadly doesn't work.

**Comment by jczhang02** [comment]
I will check it. Sorry for delay reply. I have no free time these days!

**Comment by jczhang02** [comment]
Firstly, I can't check your branch since I didn't found the `packer config` of `luasnips-mathtex-snippets`. Can u provide  a version that can be reproduced.
Moreover, the only thing u need do is that use packer to install the plugin and ensure the plugin is loaded using `PackerStatus` command.

**Comment by jczhang02** [comment]
I noticed that u are using treesitter latex parser. So that u need add `use_treesitter=true` in your config. Listed in README.
